lib/susedistribution.pm

```diff
@@ -36,6 +36,18 @@ use testapi qw(send_key %cmd assert_screen check_screen check_var click_lastmatc
   wait_still_screen wait_screen_change get_required_var diag);
 
 
+=head2 new
+
+Class constructor
+=cut
+sub new {
+    my ($class) = @_;
+    my $self = $class->SUPER::new(@_);
+
+    $self->{script_run_die_on_timeout} = 1;
```

Comment:
If i understand correct , should we set this default to 0 to keep original behavior and avoid change legacy code.

Reply:
Yes, I think so. It's better keep original behavior. If anyone need this, he/she can set to '1'

Reply:
The only sane default is to throw error when the `script_run()` command times out. There's no point running another command on the SUT when the console is still blocked by the previous one. If you have some fancy error handling that'll deal with the timeout, the correct way to preserve the old behavior is this: `script_run(.., die_on_timeout => 0);`